BloodManagmentSystem/Content/Site.css

```diff
@@ -1,24 +1,67 @@
 ﻿body {
-    padding-top: 50px;
-    padding-bottom: 20px;
+  padding-top: 50px;
+  padding-bottom: 20px;
+  font-family: 'Titillium Web', Sans-Serif;
+  font-size: 16px;
 }
-
 /* Set padding to keep content from hitting the edges */
 .body-content {
-    padding-left: 15px;
-    padding-right: 15px;
+  padding-left: 15px;
+  padding-right: 15px;
 }
-
 /* Override the default bootstrap behavior where horizontal description lists 
    will truncate terms that are too long to fit in the left column 
 */
 .dl-horizontal dt {
-    white-space: normal;
+  white-space: normal;
 }
-
 /* Set width on the form input elements since they're 100% wide by default */
 input,
 select,
 textarea {
```

Comment:
It might be good to wrap them with grid so they could be more customizable and compliant with RWD

Reply:
This one also was part of default ASP.NET MVC project